synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -445,6 +445,18 @@ export class SynopsysToolsParameter {
       const parsedUrl = url.parse(collectionUri);
       azureInstanceUrl = `${parsedUrl.protocol}//${parsedUrl.host}`;
       azureOrganization = parsedUrl.pathname?.split("/")[1] || "";
+      if (!azureOrganization) {
+        const protocolSeparator = "://";
+        const protocolIndex =
+          collectionUri.indexOf(protocolSeparator) + protocolSeparator.length;
+        const domainIndex = collectionUri.indexOf(".visualstudio.com");
```

Comment:
can we gather org name from hostname itself, we can split the hostname by dot(.) and get the info

Reply:
seems to be good idea, will fetch from hostname.